### DISCUSSION TITLE
There is a big difference between the actual synthesis time and the processing time.

### DISCUSSION THREAD

**Discussion by harmlessman** [coqui-ai/TTS#1843] [General Q&A] [answered]
If you proceed with voice synthesis in the following sentence, it takes about 8 seconds.
As shown in the picture, the `processing time` is 2.7 seconds. Why is there such a big time difference?

![image](https://user-images.githubusercontent.com/87223285/183591317-da36fe99-4b03-4aaf-9dd4-a5f11ce8e531.png)

And what does `processing time` and `real-time` mean each?
The `processing time` seems to be the time it takes to synthesize, but I don't know for sure.

`python .\TTS\bin\synthesize.py --text "" --model_path --config_path --out_path`
Synthesis was performed using this command.

**Comment by p0p4k** [comment, marked as answer]
`8 seconds = model loading time + processing time`
`real-time factor = audio_wav_time / processing_time`
real-time factor > 1 is desired.

**Reply by harmlessman** [reply]
thank you! 
I have one more Question.
Can I reduce `model loading time` more?

**Reply by p0p4k** [reply]
You can load it once, store in memory. Then just use that, unless you change the model.
See my PR #1686 , you can see an example of how I do it.

**Reply by harmlessman** [reply]
Thank you for your answer. :)
